version/K8S.VERSION

```diff
@@ -0,0 +1 @@
+v0.11.0-1.9.9
```

Comment:
Remove K8S version

Reply:
My consideration is to make it different from `PAI version`, they would be decoupled later.